client/disputes/new-evidence/style.scss

```diff
@@ -294,9 +294,6 @@
 }
 
 @media print {
-	body * {
-		visibility: hidden !important;
-	}
 	.wcpay-dispute-evidence-cover-letter,
 	.wcpay-dispute-evidence-cover-letter * {
 		visibility: visible !important;
```

Comment:
Are these lines still needed, now that the `visibility: hidden` has been removed?